README.md

```diff
@@ -240,7 +240,7 @@ We already learned about the following use cases from users:
 We have SDKs available for the following programming languages:
 
 1. Java [immudb4j](https://github.com/codenotary/immudb4j)
-2. Golang ([connection guide](https://docs.immudb.io/master/develop/connection.html), [Gorm adapter](https://github.com/codenotary/immugorm))
+2. Golang ([connection guide](https://docs.immudb.io/master/connecting/sdks.html), [Gorm adapter](https://github.com/codenotary/immugorm))
```

Comment:
just to keep a consistency with the others sdks, probably it's best to include the link to the golang sdk package
```suggestion
2. Golang ([golang sdk](https://pkg.go.dev/github.com/codenotary/immudb/pkg/client), [Gorm adapter](https://github.com/codenotary/immugorm))
```